File: app/internal/publisher/template/dotnet/widgets/WidgetEntry.cs
using System;

namespace WebsmithSDK.Widgets;

public static class WidgetEntry
{
    public static void Render()
    {
        Console.WriteLine("\n=== ${product_name} SDK Widgets ===\n");
        Console.WriteLine("Available widgets:");
        Console.WriteLine("  1. Status Widget - Check license status");
        Console.WriteLine("  2. Activation Button - Activate a license");
        Console.WriteLine("  3. Dashboard Widget - View license overview");
        Console.WriteLine("  4. Settings Widget - Configure SDK settings");
        Console.Write("\nSelect widget (1-4): ");
        var choice = Console.ReadLine()?.Trim();
        switch (choice)
        {
            case "1":
                StatusWidget.Render();
                break;
            case "2":
                ActivationButtonWidget.Render();
                break;
            case "3":
                DashboardWidget.Render();
                break;
            case "4":
                SettingsWidget.Render();
                break;
            default:
                Console.WriteLine("Invalid choice.");
                break;
        }
    }
}

File: app/internal/publisher/template/java/widgets/WidgetEntry.java
package com.websmith.sdk.widgets;

import java.util.Scanner;

public class WidgetEntry {
    public static void render() {
        System.out.println("\n=== ${product_name} SDK Widgets ===\n");
        System.out.println("Available widgets:");
        System.out.println("  1. Status Widget - Check license status");
        System.out.println("  2. Activation Button - Activate a license");
        System.out.println("  3. Dashboard Widget - View license overview");
        System.out.println("  4. Settings Widget - Configure SDK settings");
        System.out.print("\nSelect widget (1-4): ");
        Scanner scanner = new Scanner(System.in);
        String choice = scanner.nextLine().trim();
        switch (choice) {
            case "1":
                StatusWidget.render();
                break;
            case "2":
                ActivationButtonWidget.render();
                break;
            case "3":
                DashboardWidget.render();
                break;
            case "4":
                SettingsWidget.render();
                break;
            default:
                System.out.println("Invalid choice.");
                break;
        }
    }
}
